nixos/modules/services/web-apps/spliit.nix

```diff
@@ -0,0 +1,245 @@
+{
+  config,
+  lib,
+  pkgs,
+  ...
+}:
+let
+  cfg = config.services.spliit;
+in
+{
+  meta = {
+    maintainers = with lib.maintainers; [ qvalentin ];
+    doc = ./spliit.md;
+  };
+
+  options.services.spliit = with lib; {
+    enable = mkEnableOption "Spliit bill-splitting web application";
+    package = mkPackageOption pkgs "spliit" { };
+
+    port = mkOption {
+      type = types.port;
+      default = 3000;
+      description = "Port to listen on.";
+    };
```

Comment:
Use `settings.PORT` instead. You might want to use `freeformType` to specify some options.
```suggestion
```